pkg/synthetictests/updates.go

```diff
@@ -0,0 +1,20 @@
+package synthetictests
+
+import (
+	"github.com/openshift/origin/pkg/monitor"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+)
+
+func testMasterNodesUpdated(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "[sig-sippy] master nodes updated"
```

Comment:
I'd suggest sig-coreos

Reply:
Sounds good.  Noticed that https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27851/pull-ci-openshift-origin-master-e2e-aws-ovn-upgrade/1645929237531594752 did not flag the node update but it looks like one did occur.  Looks like the locator doesn't have 'master' in it so I have to verify / change my logic for detection.  I will the change to sig-coreos as well.